client/verta/verta/client.py

```diff
@@ -926,3 +956,204 @@ def endpoints(self, workspace = None):
+        parameters.
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the Experiment Run. If no name is provided, one will be generated.
+        desc : str, optional
+            Description of the Experiment Run.
+        tags : list of str, optional
+            Tags of the Experiment Run.
+        attrs : dict of str to {None, bool, float, int, str}, optional
+            Attributes of the Experiment Run.
+
+        Returns
+        -------
+        :class:`ExperimentRun`
+
+        Raises
+        ------
+        ValueError
+            If an Experiment Run with `name` already exists.
+        AttributeError
+            If an Experiment is not yet in progress.
+
+        """
+        if self._ctx.expt is None:
+            self.set_experiment()
+
+        self._ctx.expt_run = ExperimentRun._create(self._conn, self._conf, self._ctx, name, desc=desc, tags=tags, attrs=attrs, date_created=date_created)
+
+        return self._ctx.expt_run
+
+
+    def create_registered_model(self, name=None, desc=None, labels=None, workspace=None, public_within_org=None):
+        """
+        Attaches a registered_model to this Client.
+
+        A registered_model with name `name` does will be created and initialized with specified metadata parameters.
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the registered_model. If no name is provided, one will be generated.
+        desc : str, optional
+            Description of the registered_model.
+        labels: list of str, optional
+            Labels of the registered_model.
+        workspace : str, optional
+            Workspace under which the registered_model with name `name` exists. If not provided, the current
+            user's personal workspace will be used.
+        public_within_org : bool, default False
+            If creating a registered_model in an organization's workspace, whether to make this registered_model
+            accessible to all members of that organization.
+
+        Returns
+        -------
+        :class:`~verta._registry.model.RegisteredModel`
+
+        Raises
+        ------
+        ValueError
+            If a registered_model with `name` already exists.
+
+        """
+        name = self._set_from_config_if_none(name, "registered_model")
+        workspace = self._set_from_config_if_none(workspace, "workspace")
+
+        if workspace is None:
+            workspace = self._get_personal_workspace()
+
+        self._ctx = _Context(self._conn, self._conf)
+        self._ctx.workspace_name = workspace
+
+        self._ctx.registered_model = RegisteredModel._create(self._conn, self._conf, self._ctx, name, desc=desc, tags=labels, public_within_org=public_within_org)
+
+        return self._ctx.registered_model
+
+
+    def create_endpoint(self, path=None, description=None, workspace=None):
```

Comment:
Since `path` is required for `Endpoint`, I think we should require it in the function signature, even though it's different from the other `create_*()`s
```suggestion
    def create_endpoint(self, path, description=None, workspace=None):
```